README.md

```diff
@@ -12,43 +12,89 @@
+  <a href="./i18n/README.ar-dz.md">العربية</a> |
+  <a href="./i18n/README.it-it.md">Italiano</a> |
+  <a href="./i18n/README.ua-ua.md">Українська</a> |
+  <a href="./i18n/README.es-es.md">Español</a> |
+  <a href="./i18n/README.pt.md">Português</a> |
+  <a href="./i18n/README.ja-ja.md">日本語</a> |
+  <a href="./i18n/README.tr-tr.md">Türkçe</a> |
+  <a href="./i18n/README.hn-in.md">हिंदी</a> |
+  <a href="./i18n/README.vi-vn.md">Tiếng Việt</a>
+</p>
 
-QuestDB is an open-source time-series database for high throughput
-ingestion and fast SQL queries with operational simplicity.
+---
 
-QuestDB is well-suited for financial market data, IoT sensor data and real-time dashboards. It shines for datasets with [high cardinality](https://questdb.io/glossary/high-cardinality/)
-and is a drop-in replacement for InfluxDB via support for the InfluxDB Line Protocol.
+QuestDB is the fastest growing open-source time-series database.
 
-QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to filter and downsample data,
-or correlate data from multiple sources using relational and time-series joins.
+It provides **high throughput ingestion** and **fast SQL queries**.
 
-We achieve high performance by adopting a column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques.
-The entire codebase is built from the ground up in Java, C++ and [Rust](https://questdb.io/blog/leveraging-rust-in-our-high-performance-java-database/) with no dependencies and zero garbage collection.
+QuestDB is hardware efficient, with quick setup and operational efficiency.
 
-QuestDB supports schema-agnostic streaming ingestion using the InfluxDB line protocol and a REST API for bulk imports and exports.
-The QuestDB SQL Web Console is an interactive SQL editor facilitating CSV import. Finally, QuestDB also includes the Postgres Wire Protocol
-for programmatic queries.
+> Ready to go? Check out our
+> [quick start guide](https://questdb.io/docs/quick-start/).
 
-Popular tools that integrate with QuestDB include [Apache Kafka](https://questdb.io/docs/third-party-tools/kafka/questdb-kafka/), [Grafana](https://questdb.io/docs/third-party-tools/grafana/),
-[Superset](https://questdb.io/docs/third-party-tools/superset/), [Telegraf](https://questdb.io/docs/third-party-tools/telegraf/) and [Apache Flink](https://questdb.io/docs/third-party-tools/flink/).
+<p>&nbsp;</p>
 
 <div align="center">
-  <a href="https://demo.questdb.io">
-    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="600" />
+  <a href="https://demo.questdb.io/">
+    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="900" />
   </a>
+  <p><em>QuestDB Web Console</em></p>
 </div>
 
```

Comment:
Comment on the intro paragraph:

I like the refocus on simplicity, but also losing the focus on "obsession for performance" that I think should be maintained